core-collapse.html

```diff
@@ -239,10 +247,11 @@
         return;
       }
       if (!this.isTargetReady) {
-        this.targetChanged(); 
+        this.targetChanged();
       }
       this.horizontalChanged();
       this[this.opened ? 'show' : 'hide']();
+      this.notifyResize();
```

Comment:
IIUC, notifyResize() should be called whenever the size of an element is changed, or when it's shown/hidden. Since core-overlay seems to resize the element during the transition, it needs to give a notification whenever that changes.